prototype/p2p/network_mocks.go

```diff
@@ -0,0 +1,20 @@
+package p2p
```

Comment:
We can autogenerate mocks with `mockgen`. 

I already have a `make target` for it in https://github.com/pokt-network/pocket/pull/29.

This way the autogenerated code doesn't need to be submitted to master.

Reply:
I will look into mockgen